app/src/main/java/org/eyeseetea/malariacare/layout/utils/LayoutUtils.java

```diff
@@ -219,4 +261,89 @@ public static String getAppNameColorString() {
                 R.color.appNameColor);
         return String.format("%X", appNameColor).substring(2);
     }
+
+    public static void updateSurveyActionBarChartAddingQuestion(ActionBar actionBar, long surveyId, final boolean compulsory){
+    }
+
+    public static void updateSurveyActionBarChartRemovingQuestion(ActionBar actionBar, long surveyId, final boolean compulsory){
+
+    }
+
+    private static void updateChart(SurveyAnsweredRatio surveyAnsweredRatio,
+            DoublePieChart doublePieChart) {
+        doublePieChart.createDoublePie(surveyAnsweredRatio.getMandatoryStatus(),
+                surveyAnsweredRatio.getTotalStatus());
+    }
+
+    public static void updateSurveyActionBarChart(ActionBar actionBar, long surveyId){
```

Comment:
this method is used only from own class, can you refactor this method to private?